src/gbm/gbtree.cc

```diff
@@ -27,9 +28,11 @@
 #include "xgboost/host_device_vector.h"
 #include "xgboost/json.h"
 #include "xgboost/logging.h"
+#include "xgboost/model.h"
 #include "xgboost/objective.h"
 #include "xgboost/predictor.h"
-#include "xgboost/string_view.h"
+#include "xgboost/string_view.h"  // for StringView
```

Comment:
I kept the custom string view for now. Some changes in c++20 string_view might be useful, we can back-port it to xgboost when needed.